### PR TITLE
Mentor application changes

### PR DESCRIPTION
**Pull request by chamika**
https://trello.com/c/UjHF4lXv/51-as-a-mentor-application-form-update-more-detailed-qs

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -197,6 +197,10 @@
     t.text "resignation_reason"
     t.text "rematch_reason"
     t.boolean "gdpr_consent", default: false
+    t.boolean "like_mentoring_beginner", default: false
```
Comment (pg20) [inline db/schema.rb:200]: what does this field do ? Can we add comments on column that we add ?

**Comment by pg20** [comment]
Hi, can you remove this q. It is not used anymore. 
Which operating system do you feel comfortable working with during the program?

File: db/schema.rb (hunk 1 above)
Comment (chamika) [reply db/schema.rb]: Added comment

**Comment by chamika** [comment]
> Hi, can you remove this q. It is not used anymore. Which operating system do you feel comfortable working with during the program?

This is done.

**Comment by chamika** [comment]
Closing this as merging or rebasing master on top of this branch will drop some commits due to #222

**Pull request closed without merging**